master_discovery_fkie/src/master_discovery_fkie/master_discovery.py

```diff
@@ -1058,7 +1088,7 @@ def rosservice_list_masters(self, req):
         masters = list()
         with self.__lock:
             try:
-                for (_, v) in self.masters.iteritems():
+                for (k, v) in self.masters.iteritems():
```

Comment:
since I'm using the variable I wanted to remove the '_' name, which typically indicates an unused variable to get around pylint/pep8 warnings